zramsync.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=zramsync
+RequiresMountsFor=/opt/zram/zram1/upper /opt/zram/zram2/upper

```

Comment:
We would also need to dynamically add /opt/zram/zram3/upper when FIND3 gets installed

Reply:
I'm not sure how to do that, We should also sync logging as it shouldn't be too hard.

Reply:
err yes. I thought persistence and logging are #1 #2 but FIND - if installed - is #2 

We should reorder ztab and put logging first, persistence second. Then in find_install() where it creates the extra ztab line also add another line  RequiresMountsFor=/opt/zram/zram3/upper (a separate line is easier to grep out on removal).

Reply:
Can you create an issue in openHABian and assign me to it for that piece? I'll reorder the lines in ztab and then once those are done we can add those pieces in openHABian. 

Also, was there any particular reason that you have not been syncing logs?

Reply:
done: https://github.com/openhab/openhabian/issues/1100

No not to sync logs was a bug. As I wrote further up I though #2 is logs. It actually was until we added (or rather *inserted*) FIND3

Reply:
I'll make sure that is fixed as well then.